build-tools/create-pkg/create-pkg.targets

```diff
@@ -0,0 +1,92 @@
+    </PropertyGroup>
+    <RemoveDir Directories="$(PayloadDir)" />
+    <Copy
+        SourceFiles="@(FrameworkFiles)"
+        DestinationFolder="$(XBuildFrameworkDir)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(FirstFrameworkFiles)"
+        DestinationFolder="$(XBuildFrameworkDir)/$(FirstFrameworkVersion)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(BCLFrameworkFiles)"
+        DestinationFolder="$(XBuildFrameworkDir)/$(BclFrameworkVersion)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(MSBuildFiles);@(MSBuildFilesUnix)"
+        DestinationFolder="$(XBuildDir)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(MSBuildFilesDarwin)"
+        DestinationFolder="$(XBuildDir)/Darwin"
+    />
+    <Copy
+        SourceFiles="@(MSBuildLibFiles)"
+        DestinationFolder="$(XBuildDir)/lib/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(MSBuildLibFilesDarwinHost)"
+        DestinationFolder="$(XBuildDir)/lib/host-Darwin"
+    />
+    <Copy
+        SourceFiles="@(VersionFiles)"
+        DestinationFolder="$(XAFrameworkDir)"
+    />
+    <Copy
+        SourceFiles="@(TargetsFiles)"
+        DestinationFolder="$(XAFrameworkDir)/lib/xamarin.android/xbuild/%(RecursiveDir)"
+    />
+  </Target>
+  <Target Name="_CreateSymbolicLinks"
+      DependsOnTargets="_CopyFilesToPayloadDir">
+    <Exec WorkingDirectory="$(PayloadDir)/Library/Frameworks/Xamarin.Android.framework/Versions"
+      Command="ln -fs $(XAVersion) Current"
+    />
+    <MakeDir Directories="$(MonoFrameworkExternalDir)/xbuild"/>
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)/xbuild"
+      Command="ln -fs $([System.IO.Path]::GetFullPath('$(XAFrameworkDir)/lib/xamarin.android/xbuild/Novell')) ."
+    />
+    <MakeDir Directories="$(MonoFrameworkExternalDir)/xbuild/Xamarin"/>
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)/xbuild/Xamarin"
+      Command="ln -fs $([System.IO.Path]::GetFullPath('$(XBuildDir)')) ."
+    />
+    <MakeDir Directories="$(MonoFrameworkExternalDir)/xbuild-frameworks"/>
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)/xbuild-frameworks"
+      Command="ln -fs $([System.IO.Path]::GetFullPath('$(XBuildFrameworkDir)')) ."
+    />
+  </Target>
+  <Target Name="_FinalizePayload"
+      DependsOnTargets="_CreateSymbolicLinks">
+    <ReplaceFileContents
+        SourceFile="distribution.xml.in"
+        DestinationFile="distribution.xml"
+        Replacements="@PACKAGE_TITLE@=Xamarin.Android OSS $(XAVersion)">
+    </ReplaceFileContents>
+  </Target>
+  <Target Name="_PackageArchive"
+      DependsOnTargets="_FinalizePayload">
+    <RemoveDir Directories="$(PkgOutputPath)" />
+    <MakeDir Directories="$(PkgOutputPath)"/>
+    <Exec Command="pkgbuild --root $(PayloadDir) --identifier com.xamarin.android-oss.pkg --version $(XAVersion) --install-location $(PkgInstallDir) $(PkgOutputPath)/xamarin.android-oss.pkg"/>
```

Comment:
Command-line arguments which contains paths should always be quoted, in case the path contains a space.  This should use `&quot;` in the appropriate locations.